internal/clients/app_sync.go

```diff
@@ -60,6 +60,8 @@ type App struct {
 	Region        string `json:"region"`
 	IsEnx         bool   `json:"is_enx,omitempty"`
 	AndroidTarget `json:"android_target"`
+	AgencyColor   string `json:"agencyColor"`
```

Comment:
Ugh are these really camelCase when the other fields are snake case?

Reply:
This is being changed to agency_color

Reply:
fixed upstream

Reply:
❤️